cmd/kube-proxy/app/server_linux.go

```diff
@@ -112,7 +112,11 @@ func (s *ProxyServer) platformCheckSupported(ctx context.Context) (ipv4Supported
 
 	if isIPTablesBased(s.Config.Mode) {
 		// Check for the iptables and ip6tables binaries.
-		ipts := utiliptables.NewDualStack()
+		var ipts map[v1.IPFamily]utiliptables.Interface
+		ipts, err = utiliptables.NewDualStack()
+		if err != nil {
+			logger.Error(err, "Encountered errors in setting up iptables")
+		}
```

Comment:
I would merge this with the existing error handling: if neither ipv4 nor ipv6 is supported, wrap this error as part of the return value from this function. If only one of them is not supported, log the error message as part of the "no support for family" message below (keeping it as `logger.Info` since it's probably an intentional configuration rather than an error in that case)